operations/network/management_network/firmware/update_management_network_firmware.md

```diff
@@ -13,10 +14,10 @@ All firmware can be found in the HFP package provided with the Shasta release.
 
 | Model | software version |
 | ----- | -----: |
-| Aruba 8320 Switch Series | 10.09.0010  |
-| Aruba 8325 Switch Series | 10.09.0010  |
-| Aruba 8360 Switch Series | 10.09.0010  |
-| Aruba 6300 Switch Series | 10.09.0010  |
+| Aruba 8320 Switch Series | 10.11.1010  |
+| Aruba 8325 Switch Series | 10.11.1010  |
+| Aruba 8360 Switch Series | 10.11.1010  |
+| Aruba 6300 Switch Series | 10.11.1010  |
 | Mellanox SN2100 Switch Series | 3.9.3210|
 | Mellanox SN2700 Switch Series | 3.9.3210|
 | Dell S3048-ON Switch Series | 10.5.1.4|
```

Comment:
Instead of adding the numbers to the `.spelling` file, please wrap them in back-ticks. It'll make them easier to read, and the spellchecker will ignore them.

Reply:
there is already number of other number series in the spelling file, quite honestly the whole spelling file should not even touch the number series. while I am ok of changing and adding them to back ticks it to me is not the solution we should do here. 

back ticks in my opinion are just a way to NOT fix the source of the issue (which is the spelling file)

Reply:
I have another PR in progress (that has been halted for a bit) for fixing the spelling file. Unfortunately there isn't an available way for handling multi-decimal numbers like these. I do think it is easier to read IPs when they are monospaced, which is the only notion that puts me in favor of doing backticks.

Reply:
*IPs and fw version strings